medium-tests/src/test/java/mediumtest/fixtures/SonarLintTestRpcServer.java

```diff
@@ -158,6 +165,10 @@ public StorageService getIssueStorageService() {
     return serverUsingJava.getIssueStorageService();
   }
 
+  public MatchedSonarProjectBranchRepository getMatchedSonarProjectBranchRepository() {
```

Comment:
Shouldn't we avoid exposing core internals to medium tests that on the long run?

Reply:
I agree it's a shortcut. I call this method to set an initial matched branch when starting the backend, to avoid having to configure the client. It's not used from the medium tests, only from the fixtures. I will think about it, but I prefer to merge this one to avoid further conflicts